Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -6926,9 +6926,7 @@ - (void)_showKeyboard
     if (self.isFirstResponder && !_suppressSelectionAssistantReasons)
         [_textInteractionAssistant activateSelection];
 
-#if !PLATFORM(WATCHOS)
     [self reloadInputViews];
-#endif
```

Comment:
Intentional change?

Reply:
Ah, yes — (it wasn't super clear from the commit message, but I adjusted this since we no longer call `-_showKeyboard` on watchOS). I can add a sentence to the commit message to make this clearer.